### ISSUE TITLE
Bug with green flashes

### ISSUE THREAD

**Issue by LukasWallrich** [r-rust/gifski#16]
I am trying to use gifski in combination with gganimate, and get gifs that show a strange green flashing behaviour - see example below (this was also reported by many others - https://github.com/thomasp85/gganimate/issues/431). Apparently, this bug has been fixed in gifksi 1.4.0 (https://github.com/ImageOptim/gifski/issues/169), but it still occurs when I install the latest version of this package.

Is there any way to update the gifski version used by this package? Or do you think the bug is linked to recent gifski-R changes? I am on Windows and don't know how rust works ... so I am rather stuck.

![](https://i.imgur.com/OhuHFOs.gif)

**Comment by LukasWallrich** [comment]
I just installed an older gifski version (ad925fe) and it works with that ... so it probably is not about the underlying gifksi version but about the recent CRAN-induced changes ...

**Comment by jeroen** [comment]
Which operating system is this on? Can you please try to add some code with a minimal example so that I can test this.

**Comment by LukasWallrich** [comment]
I am on Windows 10 - below the code:

```
library(magrittr)
library(ggplot2)
library(gganimate)
#> Warning: package 'gganimate' was built under R version 4.0.4
anim <- mtcars %>% ggplot(aes(x = wt, y = mpg)) +
  geom_point() +
  transition_states(cyl, transition_length = 1, state_length = 2)

animate(anim, renderer = gifski_renderer())
```

**Comment by jeroen** [comment]
Thanks I can reproduce this.

**Comment by jeroen** [comment]
This should be fixed in the next CRAN release, gifski 1.4.3
